packages/filesystem/src/browser/style/file-dialog.css

```diff
@@ -87,15 +93,52 @@
     left: 41px;
 }
 
+.dialogContent .theia-NavigationUp {
+    left: 61px;
+}
+
 .dialogContent .theia-LocationListPanel {
     position: absolute;
-    left: 72px;
+    display: flex;
+    left: 82px;
     top: 1px;
+    width: 417px;
+    height: 21px;
```

Comment:
Ideally we should not hardcode values (such as height and width) but rather calculate them, or provide variables such as:

https://github.com/eclipse-theia/theia/blob/39887bf9d05c29fedc9466db76ca0a58706a1a2a/packages/core/src/browser/style/menus.css#L17-L26